rust/azure_iot_operations_services/src/leased_lock/lock.rs

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Client for Lock operations.
+
+use std::{sync::Arc, time::Duration};
+
+use crate::leased_lock::{Error, ErrorKind, lease::Client as LeaseClient};
```

Comment:
```suggestion
use crate::leased_lock::{Error, ErrorKind, lease};
```
you're not using it that many places, but I think for clarity and symmetry with how we generally use the clients that look like this, just refer to it as lease::Client in the code

Reply:
Done.